packages/hydrogen/docs/staticPages/authenticate.server.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * server.js
+ * ---------
+ * Create a storefront client.
+ * Check the server.js file in the root of your new Hydrogen project to see
+ * an example implementation of this function. If you start from an official
+ * Hydrogen template (Hello World or Demo Store), then the client is already
+ * set up for you. Update the Shopify store domain and API token to start
+ * querying your own store inventory.
+ */
+const {storefront} = createStorefrontClient({
+  cache,
+  waitUntil,
+  i18n: {language: 'EN', country: 'US'},
+  publicStorefrontToken: env.PUBLIC_STOREFRONT_API_TOKEN,
```

Comment:
It may be nice to show where `env` comes from, since it's not a global variable but people may assume that it is.